apps/opensearch_certs.sh

```diff
@@ -37,5 +37,9 @@ rm admin-key-temp.pem \
 # Store
 mv client.pem \
    client-key.pem \
-   root-ca.srl \
    root-ca-key.pem -t tls_store
+
+# openssl 3.0 workaround
+if [ $(lsb_release -sr) = 20.04 ]; then
```

Comment:
@Ark74 Please follow SC recomendations. 

Also, this is on Ubuntu 22.04, so we should really find a valid fix for that instead of doing a workaround like this. :)

Reply:
SC?

Does that mean that 20.04 is deprecated?